crates/runtime/src/http/v1/responses.rs

```diff
@@ -0,0 +1,259 @@
+use std::sync::Arc;
+
+use crate::http::{
+    traceparent::override_task_history_with_traceparent,
+    v1::chat::{KEEP_ALIVE_INTERVAL, OpenaiErrorEvent, openai_error_to_response},
+};
+use async_openai::types::responses::{
+    Content, CreateResponse, OutputContent, Response as OpenAIResponse, ResponseEvent,
+    ResponseStream,
+};
+use async_stream::stream;
+use axum::{
+    Extension, Json,
+    http::{HeaderMap, StatusCode},
+    response::{
+        IntoResponse, Response,
+        sse::{Event, KeepAlive, Sse},
+    },
+};
+use futures::StreamExt;
+use itertools::Itertools;
+use std::convert::Infallible;
+use std::time::Duration;
+use tokio::sync::RwLock;
+use tracing::{Instrument, Span};
+
+use crate::model::LLMResponsesModelStore;
+use llms::responses::Responses;
+
+fn extract_text(resp: &OpenAIResponse) -> String {
+    resp.output
+        .iter()
+        .filter_map(|out| {
+            if let OutputContent::Message(msg) = out {
+                msg.content.first().and_then(|c| {
+                    if let Content::OutputText(text) = c {
+                        Some(text.text.clone())
+                    } else {
+                        None
+                    }
+                })
+            } else {
+                None
+            }
+        })
+        .join("\n")
+}
+
+#[cfg_attr(feature = "openapi", utoipa::path(
+    post,
+    path = "/v1/chat/completions",
```

Comment:
This path needs to be updated - please also review the rest of this API documentation to see if it should be updated - it looks like this was just copied from the chat completions API.